apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/options/CreateUserOptions.java

```diff
@@ -97,7 +97,7 @@ private ServerRequest(String name, String password) {
       if (password != null)
          user.password = password;
       if (tenant != null)
-         user.tenant = tenant;
+         user.tenantId = tenant;
```

Comment:
I would rather not change the other one, as then it would be consistent to also change the getters/setters - and break backwards compatibility.